src/Views/Menu.java

```diff
@@ -108,7 +109,16 @@ private void processSelection(int selection) {
                 int rowsAdded = addNewProduct();
                 System.out.println(rowsAdded + " added.");
             }
-            case 3 -> System.out.println("Update inventory of a specific product");
+            case 3 -> {
+                try {
+                    Product product = controller.getProductById(intInput("Enter id of product to update: "));
+                    int newQty = intInput("Enter new stock quantity for " + product.getName() +" (Current qty " + product.getQty_on_hand() + "): ");
+                    int result = controller.updateStockQuantity(product.getId(), newQty);
+                    System.out.println(result > 0 ? "Quantity updated successfully!" : "Product not found!");
+                } catch (SQLException e) {
+                    System.out.println(e);
```

Comment:
switch to error message instead of stack trace to user? We should do one or the other but we should be consistent.

Reply:
yeah I see what you mean but just having the generic error message with no detail made it hard to tell what was going on while developing.

Reply:
yeah, I put them in while I was developing. We can stick to that. I just did it the other way for no particular reason. I'll just remember to switch the others back if you prefer that method.